contracts/BalanceTrackerBase.sol

```diff
@@ -196,95 +204,129 @@ abstract contract BalanceTrackerBase {
         // Get account balance
         uint256 balance = mapRequesterBalances[requester] + initAmount;
 
+        // Total requester delivery rate is number of requests coming to a selected mech
+        uint256 totalDeliveryRate = deliveryRate * numRequests;
+
```

Comment:
This is calculated as part of default batch behavior.

Reply:
It comes form the requester to a single mech, so for now the assumption is that those are identical. But things can be overwritten in the future via virtual override.